proposals/compatibility-mode.md

```diff
@@ -180,6 +180,16 @@ create cube maps that are not exactly 6 layers.
 
 **Justification**: OpenGL ES 3.1 does not support bgra8unorm-srgb textures.
 
+## 16. Disallow `textureLoad` with `texture_depth?` textures
+
+If a `texture_depth`, `texture_depth_2d_array`, or `texture_depth_cube` are used via an entry point
```

Comment:
Question I guess, is it, any entry point in the module, or is it the one used to create the pipeline?

This is a general question though - I thought I knew the answer until recently, but then found out that WGSL went the other way for features like f16.